packages/databricks-vscode/resources/whats-new/2.5.md

```diff
@@ -0,0 +1,67 @@
+<img src="./1.2/databricks-logo.png" alt="databricks-logo" width="100"/>
+
+# Databricks Extension v2.5.0
+
+## What's new?
+
+-   [Pipeline validation and partial update actions](#validation)
+-   [Pipeline events and diagnostics](#events)
+-   [Pipeline datasets](#datasets)
+-   [Quick run action](#quick-run)
+
+### <a id="validation"></a> Pipeline validation and partial update actions
+
+In previous versions, you could only trigger a “refresh all” pipeline update. Now we offer validation and partial update actions:
```

Comment:
```suggestion
In previous versions, you could only trigger a [Refresh all](https://docs.databricks.com/en/delta-live-tables/updates.html#how-dlt-updates) pipeline update. Now you can do validation and partial updates:
```
How about this way to add text treatment (instead of quotes)?

Reply:
(Also fine to just leave it with the quotes but the second sentence should be second person.